index.html

```diff
@@ -204,6 +204,18 @@ <h3 class="sub1">Multi-Post Stories</h3>
          </div>
       </div>
    </section>
+   <section class="contact">

```

Comment:
- [ ] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follows HTML best practices.

Reply:
All good now 👍🏽